library/zlib.po

```diff
@@ -38,332 +44,490 @@ msgid ""
+"allows computing a running checksum over the concatenation of several inputs.  "
+"The algorithm is not cryptographically strong, and should not be used for "
+"authentication or digital signatures.  Since the algorithm is designed for use as "
+"a checksum algorithm, it is not suitable for use as a general hash algorithm."
+msgstr ""
+"Calcule une somme de contrôle Adler-32 de *data*. (Une somme de contrôle Adler-32 "
+"est aussi fiable qu'un CRC32 mais peut être calculé bien plus rapidement.)  Le "
+"résultat produit est un entier non signé de 32-bit. Si *value* est défini, il "
+"devient la valeur initiale de la somme de contrôle ; sinon une valeur par défaut "
+"de 1 est utilisée.  Définir une *value* permet de calculer une somme de contrôle "
+"continue pendant la concaténation de plusieurs entrées.  Cet algorithme n'a "
+"aucune garantie cryptographique puissante, et ne doit pas être utiliser ni pour "
+"l'authentification, ni pour des signatures numériques. Conçu comme un algorithme "
+"de somme de contrôle, il n'est pas adapté pour une utilisation sous forme de clé "
+"de hachage générique."
 
 #: ../Doc/library/zlib.rst:44
 msgid ""
-"Always returns an unsigned value. To generate the same numeric value across "
-"all Python versions and platforms, use ``adler32(data) & 0xffffffff``."
+"Always returns an unsigned value. To generate the same numeric value across all "
+"Python versions and platforms, use ``adler32(data) & 0xffffffff``."
 msgstr ""
+"Renvoie systématiquement une value non-signée.  Pour produire systématiquement la "
+"même value à travers toutes les versions de Python et différentes plateformes, "
+"utilisez ``adler32(data) & 0xffffffff``."
 
 #: ../Doc/library/zlib.rst:52
 msgid ""
-"Compresses the bytes in *data*, returning a bytes object containing "
-"compressed data. *level* is an integer from ``0`` to ``9`` or ``-1`` "
-"controlling the level of compression; ``1`` is fastest and produces the "
-"least compression, ``9`` is slowest and produces the most.  ``0`` is no "
-"compression.  The default value is ``-1`` (Z_DEFAULT_COMPRESSION).  "
-"Z_DEFAULT_COMPRESSION represents a default compromise between speed and "
-"compression (currently equivalent to level 6). Raises the :exc:`error` "
-"exception if any error occurs."
-msgstr ""
+"Compresses the bytes in *data*, returning a bytes object containing compressed "
+"data. *level* is an integer from ``0`` to ``9`` or ``-1`` controlling the level "
+"of compression; ``1`` is fastest and produces the least compression, ``9`` is "
+"slowest and produces the most.  ``0`` is no compression.  The default value is "
+"``-1`` (Z_DEFAULT_COMPRESSION).  Z_DEFAULT_COMPRESSION represents a default "
+"compromise between speed and compression (currently equivalent to level 6). "
+"Raises the :exc:`error` exception if any error occurs."
+msgstr ""
+"Il compresse les octets contenus dans *data*, renvoie un objets en octets "
```

Comment:
"Il" looks strange here, maybe just drop it : "Compresse les octets…" is enough.